src/cci/cci_network.c

```diff
@@ -305,11 +305,10 @@ net_connect_srv (T_CON_HANDLE * con_handle, int host_id, T_CCI_ERROR * err_buf,
     {
       MUTEX_LOCK (create_ssl_mutex);
 
-      ret_value = ssl_session_init (con_handle, srv_sock_fd);
-      if (ret_value < 0)
+      if (ssl_session_init (con_handle, srv_sock_fd) < 0)
 	{
 	  MUTEX_UNLOCK (create_ssl_mutex);
-	  err_code = ret_value;
+	  err_code = CCI_ER_COMMUNICATION;
```

Comment:
it seems unrelated change with this PR. please try the following command before the merging.
```
git fetch upstream
git merge upstream/develop
```